src/components/ServiceReview/Grocery.js

```diff
@@ -0,0 +1,126 @@
+/** @jsx jsx */
+import { Fragment } from 'react';
+import { jsx } from '@emotion/core';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+
+import { Routes } from 'constants/Routes';
+import { routeWithParams } from 'lib/utils/routes';
+
+import Text from 'components/Text';
+import { TEXT_TYPE } from 'components/Text/constants';
+import Card from 'components/Card';
+import { PrimaryButton } from 'components/Button';
+
+import { styles } from './ServiceReview.styles';
+
+export const GroceryServiceReview = ({ id, handleSubmit, service }) => {
+  const history = useHistory();
+  const { t } = useTranslation();
+
+  const {
+    groceryList,
+    recurring,
+    time,
+    neighborhood,
+    crossStreet,
+    dietaryRestrictions,
+  } = service;
+
+  // todo: handle return to confirmation screen after editing
+  const handleChangeLocation = () => {
+    history.push(
+      routeWithParams(Routes.SERVICE_GROCERIES_WHERE, {
+        id,
+      }),
+    );
+  };
+  const handleChangeTime = () => {
+    history.push(
+      routeWithParams(Routes.SERVICE_GROCERIES_WHEN, {
+        id,
+      }),
+    );
+  };
+  const handleChangeList = () => {
+    history.push(
+      routeWithParams(Routes.SERVICE_GROCERIES_WHAT, {
+        id,
+      }),
+    );
+  };
+
+  const locationDetails = (
+    <Fragment>
+      <Text as="p" css={styles.capitalize}>
+        {/*  todo: map value to neighborhood name + location */}
```

Comment:
Super super super minor nit but could we preface this as `service todo`? trying to differentiate Supply from Service to quickly scan what needs to be done.

Reply:
yes!